scripts/report-references-to-old-versions.js

```diff
@@ -1,20 +1,20 @@
-import glob from 'glob';
+import { glob } from 'glob';
 import fs from 'fs/promises';
 import path from 'path';
 import github from '@actions/github';
 
 const FILES_TO_INCLUDE = '../microsoft-edge/**/*.md';
 const FILES_TO_IGNORE = [
-    // What's new/release notes articles are bound to always have version numbers in them. That's ok.
-    '../microsoft-edge/devtools/whats-new/**/*.md',
-    '../microsoft-edge/webview2/release-notes/index.md',
-    '../microsoft-edge/webview2/release-notes/archive.md',
-    '../microsoft-edge/progressive-web-apps/whats-new/*.md',
-    '../microsoft-edge/web-platform/release-notes/*.md',
-    // Experimental features also often have version numbers in them. Let's ignore them too.
-    '../microsoft-edge/devtools/experimental-features/index.md',
-    // Ignore the site-impacting-changes article, since this references older versions on purpose.
-    '../microsoft-edge/web-platform/site-impacting-changes.md'
+  // What's new/release notes articles are bound to always have version numbers in them. That's ok.
+  '../microsoft-edge/devtools/whats-new/**/*.md',
```

Comment:
does that line catch the new pattern for 143?
source files:
```
`../microsoft-edge/devtools/whats-new/2025/10/devtools-142.md`,
`../microsoft-edge/devtools/whats-new/143.md`,
```
live urls:
https://learn.microsoft.com/en-us/microsoft-edge/devtools/whats-new/2025/10/devtools-142
https://learn.microsoft.com/en-us/microsoft-edge/devtools/whats-new/143

@captainbrosset